### DISCUSSION TITLE
Render tag if condition is true

### DISCUSSION THREAD

**Discussion by LeMatosDeFuk** [laravel/framework#42716] [Ideas]
Hey, I've seen an idea in latte templating engine, where you can specify, if condition is true, then it will render a tag like in this example:

`<a href="..." n:tag-if="$clickable">Hello</a>`

prints `Hello` when $clickable is falsey
prints `<a href="...">Hello</a>` when $clickable is truthy

I think this could be a really cool feature and in blade it could look something like this:

```
@ifTag($clickable)
<a href="...">Hello</a>
@endifTag

```
This condition should be based on php function **strip_tags**, which should do the tricks in my opinion.

[Here is latte documentation](https://latte.nette.org/en/tags)

Tell me what you think about it!

**Comment by aguingand** [comment]
You can create a component like this:
```blade
{{-- views/components/link.blade.php --}}
@props([
  'disabled' => false,
])

@if($disabled)
   {{ $slot }}
@else
   <a {{ $attributes }}>{{ $slot }}</a>
@endif
```
then call
```blade
<x-link href="..." :disabled="$myCondition">
  Hello
</link>
```
